styles.css

```diff
@@ -0,0 +1,50 @@
+/* Add a black background color to the top navigation */
+.topnav {
+    background-color: #333;
+    overflow: hidden;
+    viewport-fit: top;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
```

Comment:
I kindly suggest using flexible units such as em, rem, vw, vh, %, instead of using a fixed unit like px to keep your website responsive with different viewport sizes. Moreover, make sure you know how to use different units wisely, not stick with one unit for all different properties 😃